src/guides/v2.3/comp-mgr/cli/upgrade-with-script.md

```diff
@@ -98,6 +102,7 @@ If the application fails with a  `We're sorry, an error has occurred while gener
    -  `<magento_root>/var/cache`
    -  `<magento_root>/var/page_cache`
    -  `<magento_root>/generated/code`
+   -  `<magento_root>/generated/metadata`
```

Comment:
Just have -
```suggestion
   -  `<magento_root>/generated`
```

Reply:
I think it good to show the sub-directories as knowledge sharing. Otherwise, newbies don't know what those directories do.

Reply:
Okay - that's a fair enough observation